_layouts/page.html

```diff
@@ -4,7 +4,13 @@
 {% include libraries.html %}
 <body>
     {% include navbar.html %}
+    {% if page.headerImg %}
+        {% include header.html %}
+    {% else %}
+        <div class="my-5 py-5">&nbsp;</div>
```

Comment:
What's this if-else for?

Reply:
When there's no header image, empty space for the navbar.